src/scripts/main.js

```diff
@@ -1,3 +1,243 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const thead = document.querySelector('thead');
+const body = document.querySelector('body');
+let accept = true;
+let selected;
+
+thead.addEventListener('click', (e) => {
+  const item = e.target.closest('th');
+
+  const cellIndex = item.cellIndex;
+
+  if (!item) {
+    return;
+  }
```

Comment:
I think will be better to switch the order here 
```suggestion
  if (!item) {
    return;
  }
  
  const cellIndex = item.cellIndex;
```

Reply:
But why?🤔

Reply:
Because why do you need this const if you don't have an item at all 🤔